### ISSUE TITLE
Implementation is different with paper.

### ISSUE THREAD

**Issue by HyunsuYEE** [Mercerai/EvGGS#4]
I'm matching the code with your paper. but event processing part is different with your paper. 
in the EventGaussian class's forward function, depth_estimator use the **leT = torch.cat([batch["leframe"], batch["left_voxel"]], dim=1) and riT = torch.cat([batch["reframe"], batch["right_voxel"]], dim=1)**.
But your pipeline say that there are only event voxels which are used for depth estimation 
<img width="1012" alt="image" src="https://github.com/user-attachments/assets/07f4497f-e979-4e25-bd94-2066fa8766fa">

**Comment by Mercerai** [comment]
Thanks for your attention, we made some minor modifications based on the original version. Actually, this would not badly affect the results of the framework. On the contrary, it will slightly improve the performance on real-world data based on our evaluation.
